frontend/src/components/view/loginPage.js

```diff
@@ -19,18 +19,33 @@ class LoginPage extends Component {
         this.setState({ error: ''});
     }
 
-    handleSubmit(event) {
-        event.preventDefault();
+    // handleSubmit(event) {
+    //     event.preventDefault();
 
-        if (!this.state.username) {
-            return this.setState({ error: 'Username is required'});
-        }
+    //     if (!this.state.username) {
+    //         return this.setState({ error: 'Username is required'});
+    //     }
 
-        if (!this.state.password) {
-            return this.setState({ error: 'Password is required'});
-        }
-        return this.setState({ error: 'Fake Log In Success'});
-    }
+    //     if (!this.state.password) {
+    //         return this.setState({ error: 'Password is required'});
+    //     }
+    //     return this.setState({ error: 'Fake Log In Success'});
+    // }
+
+    handleSubmit = event => {
+        event.preventDefault();
+        axios.post('https://bucketlist-builds.herokuapp.com/api/auth/login', this.state)
+          .then(res => {
+            localStorage.setItem('jwt', res.data.token);
+            this.props.history.push('/home');
```

Comment:
I like this functionality; it's helpful to the user to be redirected to a landing page or dashboard after logging in.